csharp/PlantIdApi/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "PlantIdApiKey": "IBoLNECZ5Celx3aSRoOqGBVrx9nCBBmNdhEbtmQCld7Xg6Vp8v",
```

Comment:
It is not a good practise to leave your API key in the code. It is like your password which is now shared with the whole world.
Please replace it with the following text: "-- ask for one: https://web.plant.id/api-access-request/ --"